src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -818,4 +820,96 @@ public function getConfigurator()
     {
         return $this->configurator;
     }
+
+    /**
+     * Sets types that will default to this definition.
+     *
+     * @param string[] $types
+     *
+     * @return Definition The current instance
+     */
+    public function setTypes(array $types)
```

Comment:
I would have named it `setAutowiringTypes()` to be consistent with the other methods.